### DISCUSSION TITLE
Filing PullRequests concerning the documentation

### DISCUSSION THREAD

**Discussion by marwalsch** [unoplatform/uno#7682] [General]
This is more of a meta question, but is there a public repository containing the documentation allowing to suggest improvements on the docs through pull requests? The [Improve this doc](https://github.com/unoplatform/uno-private/blob/release/stable/4.0/doc/articles/get-started-vs.md/#L1) link runs on a reef for me.

**Comment by jeromelaban** [comment]
Thanks for the report! The published documentation is indeed incorrect, we'll be fixing this soon.

The documentation is available [here](https://github.com/unoplatform/uno/tree/master/doc) in all cases.
